sentinel-core/src/main/java/com/alibaba/csp/sentinel/init/InitExecutor.java

```diff
@@ -31,6 +33,26 @@
 
     private static AtomicBoolean initialized = new AtomicBoolean(false);
 
+    /**
+     * Make sure the fastjson (if have one) no more less than 1.2.12
+     */
+    private static void checkFastjson() {
```

Comment:
Maybe checking here is not appropriate? This can be implemented as a single `InitFunc`. And I think it should not block the initialization even if the fastjson version is low (the behavior might be weird for users). We may need other approaches to inform the users.

Reply:
Discussion is under #736 and what's your opinion for that? Let's synchronize it.